nrf_modem/doc/at_interface.rst

```diff
@@ -194,27 +196,29 @@ The following snippet shows how to use the :c:func:`nrf_modem_at_cmd_async` func
    When there is a pending response, all other functions belonging to the AT API will block until the response is received in the callback function.
 
 .. note::
-   The application needs to escape characters in AT commands as it would when formatting it using :c:func:`printf`.
-   For example, the `%` needs to be escaped as `%%`.
+   The application must use escape characters in AT commands as it would when formatting it using :c:func:`printf`.
+   For example, the ``%`` character must be used with the escape character as ``%%``.
 
 Differences with nrf_modem_at_printf
 ====================================
 
 Both functions can be used to send a formatted AT command to the modem, the main difference is how the AT response is handled.
-:c:func:`nrf_modem_at_cmd` will will parse the modem AT response and return an error accordingly, in addition, it will copy the whole modem AT response to the supplied buffer.
-:c:func:`nrf_modem_at_printf` will parse the modem AT response and return an error accordingly, however, it will not make a copy of the AT response.
+The :c:func:`nrf_modem_at_cmd` function parses the modem AT response and returns an error accordingly.
+In addition, it copies the whole modem AT response to the supplied buffer.
+The :c:func:`nrf_modem_at_printf` function parses the modem AT response and returns an error accordingly.
+However, the function does not make a copy of the AT response.
 
-When interested in the result of the AT command, for example "OK" or "ERROR", use :c:func:`nrf_modem_at_printf`.
-When interested in the contents of the AT response, use :c:func:`nrf_modem_at_cmd` (or :c:func:`nrf_modem_at_scanf`).
+Use :c:func:`nrf_modem_at_printf` to know the result of the AT command, for example ``OK`` or ``ERROR``.
+Use :c:func:`nrf_modem_at_cmd` (or :c:func:`nrf_modem_at_scanf`) to know the contents of the AT response.
```

Comment:
I think the previous wording was clearer. You don't use `nrf_modem_at_printf` to know the result of a command nor you use `nrf_modem_at_cmd` to know the contents of the AT response. You use both functions when you want to send an AT command. The point is: 
You should use one, when interested in the contents of the response.
You should use the other, when interested in the result of the operation.
It's good to move `Use ...` in front of the sentence but I'd keep the original wording for the rest of it.

Reply:
Fixed